bin/build_llvm_perf.sh

```diff
@@ -2,33 +2,35 @@
 set -e
 set -x
 
-CODE_DIR=/usr/local/code/
```

Comment:
Btw, this script is for installing a special build of LLVM that allows us to use perf with the JIT modules. This isn't included in the default LLVM distros.